src/libappimage/desktop_integration.c

```diff
@@ -0,0 +1,712 @@
+#include <glib.h>
+#include <glib/gstdio.h>
+
+#include "desktop_integration.h"
+#include "appimage_handler.h"
+#include "appimage/appimage.h"
+
+extern char* vendorprefix;
```

Comment:
Global static vars... Can you declare it const at least?

Reply:
Done